components/shared/Loader.vue

```diff
@@ -91,15 +91,22 @@ export default class Loader extends Vue {
   margin: 0rem 1rem;
   width: 300px;
   min-height: 350px;
-  border: 1px solid $black;
-  box-shadow: $primary-shadow;
+
+  @include ktheme() {
+    box-shadow: theme('primary-shadow');
+    border: 1px solid theme('text-color');
```

Comment:
```suggestion
    border: 1px solid theme('border-color');
```

I would like to use `border-color` instead because there may be some differences between border and font colour in the future.